setup.py

```diff
@@ -21,7 +21,11 @@
             "pytest==2.9.1",
         ]
     },
-    dependency_links=[],
+    entry_points={
```

Comment:
that fine. though in reality the user would be importing their declarative base (which lives with their models or elsewhere) and just re-export as a fixture. might be hard to convey in an example but they mustn't e.g. create a new declarative base for the fixture or things won't work